pyproject.toml

```diff
@@ -31,7 +31,8 @@ strict = false
 ignore_missing_imports = true
 module = [
   "cdsapi",
-  "multiurl"
+  "multiurl",
+  "python-dateutil"
```

Comment:
I don't think this is needed if you have added `types-python-dateutil` to environment.yml (or ci/environment-ci.yml)

Reply:
I import the dateutil package, so somewhere I need to set it as a requirement more than the types.